Makefile

```diff
@@ -29,8 +29,7 @@ plugin-dev: generate
 # test runs the unit tests
 test: fmtcheck generate
 	go test -i $(TEST) || exit 1
-	echo $(TEST) | \
-		xargs -t -n4 go test $(TESTARGS) -timeout=60s -parallel=4
+	go test -timeout=60s -parallel=4 $(TEST) $(TESTARGS)
```

Comment:
We need the `xargs -n4` because the full test suite is too large for a single process in Travis.

Reply:
Hmm... interesting. Okay. Will replace this with something generally equivalent to what was there before, then.

Reply:
This is now done, and indeed it looks like the Travis tests are passing again.